<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>438</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EndWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,3,2,1">
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>105</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../../../ROOT.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="game_label">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>27</pointsize>
           <italic>true</italic>
           <fontweight>ExtraBold</fontweight>
          </font>
         </property>
         <property name="text">
          <string>✧˖°. RootRoulette ✧˖°.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="roulette_img">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../../../roulette1.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="yourscore_label">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>23</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Your score is:       </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="finalscore_label">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>40</pointsize>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Score</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>⋆.˚˖࿔ ࣪</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_button">
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>15</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Check your stats!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>16</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>࣪ ࿔˖˚.⋆</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>123</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thank_label">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>15</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Thank you for playing!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="again_button">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>130</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Try again</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
